framework/set/resources/providers/aws/createResources.go

```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// CreateAWSResources is a helper function that will create the AWS resources needed for the RKE2 cluster.
-func CreateAWSResources(file *os.File, newFile *hclwrite.File, tfBlockBody, rootBody *hclwrite.Body, terraformConfig *config.TerraformConfig,
+// CreateResources is a helper function that will create the AWS resources needed for the RKE2 cluster.
+func CreateResources(file *os.File, newFile *hclwrite.File, tfBlockBody, rootBody *hclwrite.Body, terraformConfig *config.TerraformConfig,
```

Comment:
Can you revert the name back in the function and comment? For Linode, I have it as `CreateLinodeResources`; I think we should just add the node provider name to make it easier to read.

The same for similar files below.